Marlin/thermistortables.h

```diff
@@ -979,6 +979,40 @@ const short temptable_60[][2] PROGMEM = {
   {1008 * OVERSAMPLENR, 0},
 };
 #endif
+
+#if (THERMISTORHEATER_0 == 66) || (THERMISTORHEATER_1 == 66) || (THERMISTORHEATER_2 == 66) || (THERMISTORHEATER_3 == 66) || (THERMISTORBED == 66)
```

Comment:
Thermistor 66 should be added to the "Thermal Settings" section in `Configuration.h` where it lists all the thermistor numbers.